MediaWikiTelegramBot.js

```diff
@@ -284,7 +293,38 @@ function showCurrentMwMessage(userID) {
 function advanceMwMessage(userID) {
     const user = getUser(userID);
 
+    // Make sure there is another message to translate
+    if( user.currentMwMessageIndex + 1 >= user.loadedMwMessages.length ) {
+        user.currentMwMessageIndex = 0;
+        user.loadedMwMessages = [];
```

Comment:
Well, for example, initUser is similar. But it must be done carefully. Not the most immediately important thing. And I'm changing the whole user loading and initialization anyway, in order to start using a real persistent database.

Consider it just a "thinking-out-loud" comment.